types/bosch/parkingLotSensor/schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$id": "https://akenza.io/bosch/parkingLotSensor.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Bosch Parking Lot Sensor",
+  "type": "object",
+  "properties": {
+    "occupancy": {
+      "description": "Space occupancy. 0 = Unoccupied / 1 = Occupied.",
+      "type": "integer",
```

Comment:
better use a boolean here?

Reply:
We always use int so boolean would be a little bit out of the standard